cylc/flow/scheduler.py

```diff
@@ -1050,6 +1049,133 @@ def kill_tasks(
 
         return len(unkillable)
 
+    def remove_tasks(
+        self, items: Iterable[str], flow_nums: Optional['FlowNums'] = None
+    ) -> None:
+        """Remove tasks from the task pool (by command).
+
+        Args:
+            items: Relative IDs or globs.
+            flow_nums: Flows to remove the tasks from. If empty or None, it
+                means 'all'.
+        """
+        active, inactive, _unmatched = self.pool.filter_task_proxies(
```

Comment:
```suggestion
        active, inactive, _= self.pool.filter_task_proxies(
```

Reply:
> Or is unmatched meant to be descriptive?

Yes

Reply:
This is fine. It is a Python convention to prefix unused variables with an underscore, or just leave an underscore.

This convention is inherited from functional languages where underscore variables may have a functional meaning (i.e. the variable is truly *ignored*). Python isn't a pure functional language like these, so the underscore is just a convention, it doesn't do anything and the unwanted variable is still assigned.